### ISSUE TITLE
Display transcript on Aspine

### ISSUE THREAD

**Issue by psvenk** [Aspine/aspine#100]
Aspen includes a "transcript" feature under the "My Info" tab which displays a record of the student's final grades in all classes for all school years. It would be nice if Aspine could display this information in addition to the detailed information for the current year (and previous year(s) after #99 and/or #74 is/are implemented)

**Comment by psvenk** [comment]
Duplicate of #54.
